platform/android/src/ch/opengis/qfield/QFieldProjectActivity.java

```diff
@@ -54,8 +66,62 @@ public void onCreate(Bundle bundle) {
         drawView();
     }
 
-    private void drawView() {
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        if (!getIntent().hasExtra("path")) {
+            getMenuInflater().inflate(R.menu.project_menu, menu);
+            MenuCompat.setGroupDividerEnabled(menu, true);
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+        case R.id.import_dataset: {
+            Intent intent = new Intent(Intent.ACTION_OPEN_DOCUMENT);
+            intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION |
+                            Intent.FLAG_GRANT_WRITE_URI_PERMISSION);
+            intent.addFlags(Intent.FLAG_GRANT_PREFIX_URI_PERMISSION);
+            intent.addFlags(Intent.FLAG_GRANT_PERSISTABLE_URI_PERMISSION);
+            intent.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, true);
+            intent.setType("*/*");
+            startActivityForResult(intent, R.id.import_dataset);
+            return true;
+        }
+        case R.id.import_project_folder: {
+            Intent intent = new Intent(Intent.ACTION_OPEN_DOCUMENT_TREE);
+            intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION |
+                            Intent.FLAG_GRANT_WRITE_URI_PERMISSION);
+            intent.addFlags(Intent.FLAG_GRANT_PREFIX_URI_PERMISSION);
+            intent.addFlags(Intent.FLAG_GRANT_PERSISTABLE_URI_PERMISSION);
+            startActivityForResult(intent, R.id.import_project_folder);
+            return true;
+        }
+        case R.id.import_project_archive: {
+            Intent intent = new Intent(Intent.ACTION_OPEN_DOCUMENT);
+            intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION |
+                            Intent.FLAG_GRANT_WRITE_URI_PERMISSION);
+            intent.addFlags(Intent.FLAG_GRANT_PREFIX_URI_PERMISSION);
+            intent.addFlags(Intent.FLAG_GRANT_PERSISTABLE_URI_PERMISSION);
+            intent.setType("application/zip");
+            startActivityForResult(intent, R.id.import_project_archive);
+            return true;
+        }
+        case R.id.usb_cable_help: {
+            String url = "https://qfield.org/docs/";
```

Comment:
Does this need a more precise link?

Reply:
@m-kuhn , yes, planning to talk to Lucie about this over the week, in the meantime, it's a "dummy" string to move things forward.

Reply:
She's on leave currently, best to talk to zsanett

Reply:
@m-kuhn , will do.

Reply:
Wait until we migrate the docs I guess. @marioba